tests/invoke/invoke_kernel_params.cpp

```diff
@@ -72,8 +72,12 @@ class test_kernel_name;
 class TEST_NAME : public util::test_base {
  public:
   typedef sycl::accessor<uint32_t, 1, sycl::access_mode::write,
-                             sycl::target::device>
-      acc_uint32_t;
+<<<<<<< HEAD
```

Comment:
Merge conflict marker

Reply:
Thanks for the pay attention, I've removed this in this commit: f6cd54a48b76cbfb3cd2ad739c05ed42d43dcd3c